src/Factor.jl

```diff
@@ -79,16 +79,9 @@ end
     getindex(a::Fac, b) -> Int
 
 If $b$ is a factor of $a$, the corresponding exponent is returned. Otherwise
-an error is thrown.
+return zero.
```

Comment:
Of course we *did* document that an error is raised... so strictly speaking this *is* a breaking change *sigh*